examples/compressible/dry_bryan_fritsch.py

```diff
@@ -66,14 +66,11 @@
 Vu_DG1 = VectorFunctionSpace(mesh, VDG1.ufl_element())
 Vu_CG1 = VectorFunctionSpace(mesh, "CG", 1)
 
-u_opts = RecoveryOptions(embedding_space=Vu_DG1,
-                         recovered_space=Vu_CG1,
-                         boundary_method=BoundaryMethod.taylor)
-rho_opts = RecoveryOptions(embedding_space=VDG1,
-                           recovered_space=VCG1,
-                           boundary_method=BoundaryMethod.taylor)
-theta_opts = RecoveryOptions(embedding_space=VDG1,
-                             recovered_space=VCG1)
+recovery_spaces = RecoverySpaces(domain)
```

Comment:
boundary methods are required for this example

Reply:
I think we also want `use_vector_spaces=True`